<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Start_W</class>
 <widget class="QDialog" name="Start_W">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>294</width>
    <height>361</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="Open_button">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>272</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Open exist....</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>219</y>
     <width>371</width>
     <height>31</height>
    </rect>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="NameNewDB_TexLine">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>241</y>
     <width>270</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="Create_Button">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>311</y>
     <width>272</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Create new DB </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="PassNewDB_TexLine">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>276</y>
     <width>270</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Path_line">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>145</y>
     <width>241</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="Brows_Button">
   <property name="geometry">
    <rect>
     <x>252</x>
     <y>145</y>
     <width>31</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
